hail/python/hail/experimental/sparse_split_multi.py

```diff
@@ -102,6 +105,9 @@ def struct_from_min_rep(i):
                                   alleles=[mr.alleles[0], mr.alleles[1]],
                                   a_index=i,
                                   was_split=True))
+                        .when(filter_changed_loci,
```

Comment:
was going to complain about turning a compile-time flag into a runtime check, but this will get optimized.